src/controller/java/BUILD.gn

```diff
@@ -146,11 +146,16 @@ if (chip_link_tests) {
 kotlin_library("tlv") {
   output_name = "libCHIPTlv.jar"
 
-  deps = [ "${chip_root}/third_party/java_deps:protobuf-java" ]
+  deps = [
+    "${chip_root}/third_party/java_deps:gson",
```

Comment:
this adds a json dependency into a  TLV library. This seems off: I understand a tlv/json converter to depend on json and tlv, but not bundling everything into one.

Reply:
also wondering why the previous tlv implementation was depending on protobuf ...

Reply:
Protobuf seems because `com.google.protobuf.ByteString` usage.
I believe we should move to ByteArray/String instead (as needed). We can do that separately though.

Reply:
@andy31415  I've isolated Json-Tlv convertor implementation into separate package "jsontlv".

I'll remove dependency on the protobuf as a followup PR